products/nextjs.md

```diff
@@ -29,7 +35,7 @@ releases:
 
 -   releaseCycle: "13"
     releaseDate: 2022-10-25
-    eol: false
+    eol: 2024-09-17
```

Comment:
I think we need to change line 22 (maybe `releaseDate(x+1) + 1yr`?)

relates to https://github.com/endoflife-date/endoflife.date/pull/4240

Reply:
The pattern is from the previous entries: eol(12..9) = latestReleaseDate(12..9)
As long there is no documentation from nextjs I would continue using this pattern.

Reply:
yeah, let's just ship it update it as we go 👍